ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js

```diff
@@ -519,15 +519,18 @@ export default class ConfirmApproveContent extends Component {
     }
 
     return (
-      <span
-        className="confirm-approve-content__approval-asset-title"
-        onClick={() => {
-          copyToClipboard(tokenAddress);
-        }}
-        title={tokenAddress}
-      >
-        {titleTokenDescription}
-      </span>
+      <>
+        <span
+          className="confirm-approve-content__approval-asset-title"
+          onClick={() => {
+            copyToClipboard(tokenAddress);
+          }}
+          title={tokenAddress}
+        >
+          {titleTokenDescription}
+        </span>
+        {tokenIdWrapped && <span>{tokenIdWrapped}</span>}
```

Comment:
I think we want to check for the length of `tokenIdWrapped` here since it will always be a string

Reply:
an empty string (as `tokenIdWrapped` is set if `tokenId` is undefined) is falsy though so this won't be rendered.

Reply:
oops, yes you're right